### DISCUSSION TITLE
EPG simulator

### DISCUSSION THREAD

**Discussion by kareemelmasry** [iptv-org/community#458] [Q&A]
### Description

Hii.
Is there any app can be installed to see and test the EPG files or like a simulator?

**Comment by HNJAMeindersma** [comment]
You might be able to find something here: https://github.com/iptv-org/awesome-iptv/blob/master/README.md

Also: https://letmegooglethat.com/?q=xmltv+file+check
- http://www.a123.dk/subpages/validate.html (use at your own risk!)
- https://wiki.xmltv.org/index.php/XmltvValidation (use at your own risk!)
